tests/Wallabag/ApiBundle/Controller/ConfigRestControllerTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Tests\Wallabag\ApiBundle\Controller;
+
+use Tests\Wallabag\ApiBundle\WallabagApiTestCase;
+
+class ConfigRestControllerTest extends WallabagApiTestCase
+{
+    public function testGetConfig()
+    {
+        $this->client->request('GET', '/api/config.json');
+        $this->assertSame(200, $this->client->getResponse()->getStatusCode());
+
+        $content = json_decode($this->client->getResponse()->getContent(), true);
+
+        $this->assertArrayHasKey('id', $content);
+        $this->assertArrayHasKey('items_per_page', $content);
+        $this->assertArrayHasKey('language', $content);
+        $this->assertArrayHasKey('reading_speed', $content);
+        $this->assertArrayHasKey('action_mark_as_read', $content);
+        $this->assertArrayHasKey('list_mode', $content);
+
+        $this->assertSame(200.0, $content['reading_speed']);
+        $this->assertSame('en', $content['language']);
```

Comment:
I think it could be useful to check that there is no other properties in the resulting array. In this case we could check that it only contains 6 keys.